llm/core/pypto_executor.py

```
@@ -398,6 +799,422 @@ def _project_logits(self, model: RuntimeModel, hidden: torch.Tensor) -> torch.Te
+                sm_sw["w_gate"],
+                sm_sw["w_up"],
+                sm_sw["w_down"],
+                _has_prefill_tensor,      # has_prefill = True
+                prefill_out,              # prefill_out
+                decode_out,               # decode_out
+                rms_x,                    # rms_x  (shares storage with decode_out)
+                rms_gamma,                # final_norm_weight
+                rms_normed,               # rms_normed
+                lm_head_weight,           # lm_head_weight_t
+                logits_padded,            # logits_padded
+            ]):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)